src/applications/static-pages/cta-widget/ctaWidgets.js

```diff
@@ -312,12 +312,12 @@ export const ctaWidgetsLookup = {
   },
   [CTA_WIDGET_TYPES.VIEW_APPOINTMENTS]: {
     id: CTA_WIDGET_TYPES.VIEW_APPOINTMENTS,
-    deriveToolUrlDetails: authenticatedWithSSOe => ({
```

Comment:
Given all authentication in 2025 should be single-sign-on, and we don't want people going to MHV National Portal, this change is needed.